lighthouse-cli/run.js

```diff
@@ -27,12 +27,22 @@ const _PROTOCOL_TIMEOUT_EXIT_CODE = 67;
 
 /**
  * exported for testing
- * @param {string} flags
+ * @param {string|Array<string>} flags
  * @return {Array<string>}
  */
 function parseChromeFlags(flags = '') {
-  const parsed = yargsParser(
-      flags.trim(), {configuration: {'camel-case-expansion': false, 'boolean-negation': false}});
+  const trimmedFlags = (Array.isArray(flags) ?
+      // Allow multiple --chrome-flags
+      // i.e. `lighthouse --chrome-flags="--user-agent='My Agent'" --chrome-flags="--headless"`
+      flags.join(' ') :
+      // Remove wrapping single or double quotes
+      // i.e. `lighthouse --chrome-flags="--headless --user-agent='My Agent'"
```

Comment:
want to use the real example we care about for `execFile`? AFAIK this will already work in most shells

Reply:
Yes that's correct it is only happening because of the argument escaping from execFile - I adjusted the comment